### DISCUSSION TITLE
I would like to install hackintosh

### DISCUSSION THREAD

**Discussion by johanleon98** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#41] [Q&A]
Hello, I have been searching various sites and guides on the internet but I can't run macos on my device, I found this blog and it has the same components as my laptop, I would like to know if you could explain to me a little more in detail how to install hakintosh.
My laptop is an asus rog zephyrus g14 ga401qm, with
Ryzen 7 5800HS with Radeon Graphics
16GB RAM
INTEL SSD PEKNU512GZ
NVIDIA GeForce RTX 3060

I would like to know if you could help me. Thanks for post.

**Comment by PIut02** [comment]
These two sites may be helpful to you, read them and the readme of this repository, if you still have any other questions you can ask here.
https://dortania.github.io/OpenCore-Install-Guide/installation/installation-process.html#installation-process
https://chefkissinc.github.io/guide

**Reply by johanleon98** [reply]
Thank you, I was reading the links that you shared with me, I already have MacOS Sonora downloaded, I have a question, with the EFI file that you share I could use it by making the edits to generate the keys and the model information, when installing it remains the apple logo for a moment and then the pc restarts, I think it is because (disable the NootedRedcontroller in config.plist) using your file, should I do this?

**Reply by PIut02** [reply]
Try disabling SSDT-NoHybGfx.aml and try again. There are reports that this will cause crashes but the cause is unknown.
